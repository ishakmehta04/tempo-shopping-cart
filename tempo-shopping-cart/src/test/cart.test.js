// import Cart from '../components/Cart';
// import React from 'react';
const Cart = require('../components/Cart')
// const spy = jest.spyOn(props, 'fetchRemoveItem');
// let handleRemove = () => fetchRemoveItem();

 describe('test handleRemove', function() {
     it('should call fetchRemoveItem function', function() {
        //  let id = 1;
        handleRemove();
        expect(spy).toHaveBeenCalled();
     });
    it('renders without crashing', () => {
        expect(JSON.stringify(Cart)).toMatchSnapshot();
      });
 })