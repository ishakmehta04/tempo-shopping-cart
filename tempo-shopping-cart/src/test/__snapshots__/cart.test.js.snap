// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test handleRemove renders without crashing 1`] = `undefined`;
